Event Locales & Yacht Heist Alert

NoobySloth opened the pull request:
Added English locales for cl_events and cl_extraalerts

NoobySloth committed:
Event Locales

Added English locales for cl_events and cl_extraalerts

NoobySloth committed:
Yacht Heist Alert

Added a yacht heist alert

NoobySloth committed:
Added server exports

Added what Hyper has done as well as the YachtHeist + Car Jacking

Co-Authored-By: Hyper <50504920+itsHyper@users.noreply.github.com>

MonkeyWhisper reviewed (dismissed):
How come you removed carjack? All else looks good.

NoobySloth committed:
vehicletheft dupe with carjacking, updated with main branch

Removed vehicle theft as it's the same as car jacking just a different name.
Removed main branch conflicts
Added locales

NoobySloth committed:
Remove vehicle theft dupe

NoobySloth committed:
Added command info

NoobySloth committed:
removed a blank

NoobySloth committed:
Merge branch 'main' into main

NoobySloth committed:
Removed vehicle export, the same as carjack??

NoobySloth committed:
re-added vehicle theft with locales

NoobySloth committed:
accidently left a trigger in

NoobySloth committed:
Updated vehicle shots from being "undefined" and removed blanks in dispatch codes

NoobySloth committed:
updated to latest branch

NoobySloth committed:
client trigger for rcore_gangs

probably not needed but haven't tested server sided exports

NoobySloth committed:
explosion in threads

NoobySloth committed:
Merge branch 'main' into main
